packages/web-components/fast-element/tsconfig.json

```diff
@@ -1,16 +1,20 @@
 {
     "extends": "../../../tsconfig.json",
     "compilerOptions": {
-        "outDir": "dist",
-        "baseUrl": ".",
+        "declarationDir": "dist/dts",
+        "outDir": "dist/esm",
         "strictNullChecks": true,
-        "target": "es2015"
+        "target": "es2015",
+        "module": "ESNext",
+        "removeComments": true,
+        "types": [
```

Comment:
Curious why you're needing to manually enumerate the `types` field here - are these not found by default in the `@types` dir?

Reply:
Both of these types introduce globals, so they are needed here. Removing them causes build errors for either the tests or the test setup code.